insight/tasks/weekly_newsletter_batch.py

```
@@ -0,0 +1,467 @@
+            # 뉴스레터 상태 저장
+            self.weekly_info = {
+                'newsletter_order': weekly_trend['id'],
+                's_date': weekly_trend['week_start_date'],
+                'e_date': weekly_trend['week_end_date'],
+            }
+        
+            context = to_dict(WeeklyTrendContext(insight=parse_json(weekly_trend['insight'])))
+            weekly_trend_html = render_to_string('insights/weekly_trend.html', context)
+
+            # 템플릿 렌더링이 제대로 되지 않은 경우
+            if not '이 주의 트렌딩 글' in weekly_trend_html or not '트렌드 분석' in weekly_trend_html:
```

Flake8 (7.2.0)

[error] test for membership should be 'not in'

(E713)